ocis-phoenix/CHANGELOG.md

```diff
@@ -0,0 +1,373 @@
+
+## Details
+
+* Change - Enable Settings and Accounts apps by default: [#80](https://github.com/owncloud/ocis/ocis-phoenix/pull/80)
+
+   The default ocis-web config now adds the frontend of ocis-accounts and ocis-settings to the
+   builtin web config.
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/80
+
+* Change - Update Phoenix: [#79](https://github.com/owncloud/ocis/ocis-phoenix/pull/79)
+
+   Updated phoenix from v0.14.0 to v0.15.0
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/79
+   https://github.com/owncloud/phoenix/releases/tag/v0.15.0
+
+# Changelog for [0.11.0] (2020-08-17)
+
+The following sections list the changes in ocis-phoenix 0.11.0.
+
+[0.11.0]: https://github.com/owncloud/ocis/ocis-phoenix/compare/v0.10.0...v0.11.0
+
+## Summary
+
+* Bugfix - Exit when assets or config are not found: [#76](https://github.com/owncloud/ocis/ocis-phoenix/pull/76)
+* Bugfix - Build docker images with alpine:latest instead of alpine:edge: [#73](https://github.com/owncloud/ocis/ocis-phoenix/pull/73)
+* Change - Hide searchbar by default: [#116](https://github.com/owncloud/product/issues/116)
+* Change - Update Phoenix: [#78](https://github.com/owncloud/ocis/ocis-phoenix/pull/78)
+
+## Details
+
+* Bugfix - Exit when assets or config are not found: [#76](https://github.com/owncloud/ocis/ocis-phoenix/pull/76)
+
+   When a non-existing assets folder is specified, there was only a warning log statement and the
+   service served the builtin assets instead. It is safe to exit the service in such a scenario,
+   instead of serving other assets than specified. We changed the log level to `Fatal` on
+   non-existing assets. Similar for the web config, it was not failing on service level, but only
+   showing an error in the web ui, wenn the specified config file could not be found. We changed the
+   log level to `Fatal` as well.
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/76
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/77
+
+* Bugfix - Build docker images with alpine:latest instead of alpine:edge: [#73](https://github.com/owncloud/ocis/ocis-phoenix/pull/73)
+
+   ARM builds were failing when built on alpine:edge, so we switched to alpine:latest instead.
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/73
+
+* Change - Hide searchbar by default: [#116](https://github.com/owncloud/product/issues/116)
+
+   Since file search is not working at the moment we decided to hide the search bar by default.
+
+   https://github.com/owncloud/product/issues/116
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/74
+
+* Change - Update Phoenix: [#78](https://github.com/owncloud/ocis/ocis-phoenix/pull/78)
+
+   Updated phoenix from v0.13.0 to v0.14.0
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/78
+
+# Changelog for [0.10.0] (2020-07-17)
+
+The following sections list the changes in ocis-phoenix 0.10.0.
+
+[0.10.0]: https://github.com/owncloud/ocis/ocis-phoenix/compare/v0.9.0...v0.10.0
```

Comment:
Codacy found an issue: [[final-definition] Move definitions to the end of the file (after the node at line `367`)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)